jest.js

```diff
@@ -0,0 +1,15 @@
+const pluginJest = require('eslint-plugin-jest');
+
+// Configuration for Jest test files only.
+module.exports = [
+    {
+        files: ['**/*.spec.*', '**/*.test.*', '**/test/**', '**/tests/**'],
+        ...pluginJest.configs['flat/recommended'],
+        rules: {
+            // Force at least one expect() in each test.
+            'jest/expect-expect': 'error',
+            // Disallow disabled tests.
+            'jest/no-disabled-tests': 'error',
+        }
```

Comment:
Will this override just those two rules you change here, or will it overwrite the whole `rules` object? I think you'll need to do something like this, but I'm not sure.
```suggestion
        rules: {
            ...pluginJest.configs['flat/recommended'].rules,
            // Force at least one expect() in each test.
            'jest/expect-expect': 'error',
            // Disallow disabled tests.
            'jest/no-disabled-tests': 'error',
        }
```

Reply:
yes, you're right. we wouldn't need it if we added the recommended config to the top-level array, then our overrides would be executed independently.

But in the [docs](https://github.com/jest-community/eslint-plugin-jest), they have it as follows:
<img width="666" alt="image" src="https://github.com/user-attachments/assets/cd5c1062-50a3-46cb-aa95-44b9beaebc50" />
So I'd keep it that way, thanks for noticing.